samples/apply-blend-renderer-to-hillshade/src/main/java/com/esri/arcgismaps/sample/applyblendrenderertohillshade/screens/ApplyBlendRendererToHillshadeScreen.kt

```diff
@@ -0,0 +1,211 @@
+ * and color ramp preset. Changes are applied live to the ViewModel which updates
+ * the renderer on the map.
+ */
+@Composable
+fun ApplyBlendRendererToHillshadeScreen(sampleName: String) {
+    val viewModel: ApplyBlendRendererToHillshadeViewModel = viewModel()
+
+    // Dialog visibility state
+    var isDialogOptionsVisible by remember { mutableStateOf(false) }
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        floatingActionButton = {
+            if (!isDialogOptionsVisible) {
+                FloatingActionButton(
+                    modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { isDialogOptionsVisible = true }
+                ) { Icon(Icons.Filled.Settings, contentDescription = "Show options") }
+            }
+        }
+    ) { innerPadding ->
+        Column(modifier = Modifier
+            .fillMaxSize()
+            .padding(innerPadding)) {
+
+            // MapView shows the map from the ViewModel
+            MapView(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .weight(1f),
+                arcGISMap = viewModel.arcGISMap
+            )
+
+            // Settings dialog
+            if (isDialogOptionsVisible) {
+                DialogOptions(
+                    altitude = viewModel.altitude,
+                    onAltitudeChange = viewModel::updateAltitude,
+                    azimuth = viewModel.azimuth,
+                    onAzimuthChange = viewModel::updateAzimuth,
+                    slopeTypeOptions = ApplyBlendRendererToHillshadeViewModel.slopeTypeOptions,
+                    selectedSlopeType = viewModel.slopeType,
+                    onSlopeTypeSelected = viewModel::updateSlopeType,
+                    colorRampPresets = viewModel.colorRampPresets,
+                    selectedColorRampIndex = viewModel.selectedColorRampPresetIndex,
+                    onColorRampSelected = viewModel::updateColorRampPresetIndex,
+                    onDismissRequest = { isDialogOptionsVisible = false }
+                )
+            }
+
+            // Message dialog for errors surfaced by the ViewModel
+            viewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    }
+}
+
+@Composable
+private fun DialogOptions(
+    altitude: Double,
+    onAltitudeChange: (Double) -> Unit,
+    azimuth: Double,
+    onAzimuthChange: (Double) -> Unit,
+    slopeTypeOptions: List<Pair<String, com.arcgismaps.raster.SlopeType?>>,
+    selectedSlopeType: com.arcgismaps.raster.SlopeType?,
+    onSlopeTypeSelected: (com.arcgismaps.raster.SlopeType?) -> Unit,
```

Comment:
Add the import `import com.arcgismaps.raster.SlopeType`: 
```suggestion
    slopeTypeOptions: List<Pair<String, SlopeType>>,
    selectedSlopeType: SlopeType,
    onSlopeTypeSelected: (SlopeType) -> Unit,
```